kafka-ui-react-app/src/components/common/NewTable/Table.tsx

```diff
@@ -194,6 +199,21 @@ const Table: React.FC<TableProps<any>> = ({
     return undefined;
   };
 
+  const handleRowHover = (row: Row<typeof data>) => (e: React.MouseEvent) => {
+    if (onRowHover) {
+      e.stopPropagation();
+      return onRowHover(row);
+    }
+
+    return undefined;
```

Comment:
i don't think we need this return.

Reply:
Well, I think we need it for code consistency, the handler returns `MouseEvent` that can be undefined, just like the click handler